autocomplete/zsh_autocomplete

```diff
@@ -1,3 +1,5 @@
+#compdef $PROG
```

Comment:
Hey @gnowxilef, can you please explain a bit how this fixes the ZSH completion?

Reply:
sure, on the zsh documentation page http://zsh.sourceforge.net/Doc/Release/Completion-System.html#Initialization under the "Autoloaded files" section (20.2.2)7, it states that adding `#compdef name` will make the completion file autoloadable. the current behavior of the cli does not load anything into zsh.

Reply:
Thanks 🙏